jetson/power_logging/run_experiment.sh

```diff
@@ -41,6 +41,7 @@ do
   # Run the measure_inference_power.py script
   python measure_inference_power.py \
     --model "$model" \
+    --model-repo "pytorch/vision:v0.10.0" \
```

Comment:
Is this a required parameter? Can this not be set as a default value in the script?